MAINTAINERS.yml

```diff
@@ -1104,6 +1104,18 @@ hawkBit:
     labels:
         - "area: hawkBit"
 
+UpdateHub:
```

Comment:
isn't this considered an instance of the management subsystem and area and not an area on its own? The file should not go into the level of instances os subsystems, this needs to be handles some other way or it will become unmanageable.

Reply:
Hi @nashif ,

Yes, I understand UpdateHub is part of `subsys:mgmt`. It seems that all subsys don't have a common root inside maintainers file. I noted that "MCU Manager", "OSDP" and "hawkBit" are "out of scope" too and they should be part of `subsys:mgmt`.

Should I follow same principles from `drivers` to create the level of instances? I mean:

"Subsys: mgmt: UpdateHub": ?

Reply:
Yes, I think having mgmt: updatehub would be a good idea, so we can then place all the other management subsystems under `mgmt`. OK @nashif?